techworkersco/twc-site-nl
# Resources!

The start of a resources section on our site.

Some moots/notes:

- `resources` or `articles`? (The latter is used by our ber folks)
- I assume we want to use the `/admin` interface too (It works! see #40)
- It's currently "disabled" in that there is no reference link in the Menu, however commenting back in the header_themes value for resources makes it available. The links are there and ready to use, but you just need to hack them into the URL manually for now until we're ready to expose it all etc.
- IMO, If there is no translation, we should just remove the if/else condition and just keep it in one language. We could eventually look at an include for "Help us by commiting changes/translation in our github" or something.

:warning: There is a mild problem with permalinks. If `permalink_nl` isn't specified, it will try to render into `/nl/resources/` (which in theory should never exist. The selection in the NL bronnen page will likely still work, however the language selector won't.

## Tests

From the git repo:

1. Copy the template to a file without the prefixed "_"

```bash
cp _resources/_template.md _resources/testing.md
```

2. Edit some of the content - Be amazed at how it "just works"
3. Check it out in the dockers/jekyll directly
